### PR TITLE
External webbings

### PR DESCRIPTION
**Pull request by Dorodomki**
# About the pull request

Makes the External Webbing Work like an Hazard suit, giving it more utility and storage variety

# Explain why it's good for the game

it makes play forecon survivor a bit less horrible and makes the external webbing be at the same level with an hazard suit

# Changelog
:cl: 
add: External webbing now allows more items to be stored in it (much like the hazard suit).
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -276,15 +276,36 @@
 	desc = "Designed to be worn over a jumpsuit rather than clipped on."
 	icon_state = "webbing"
 	item_state = "webbing"
-	allowed = list(
+
+	allowed = list (
+		/obj/item/device/analyzer,
+			/obj/item/device/multitool,
```
Comment (harryob) [inline code/modules/clothing/suits/miscellaneous.dm:282]:
```suggestion
		/obj/item/device/multitool,
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by morrowwolf** [comment]
I believe the no guns may be intentional.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Remove from staff officer lockers and you're good to go.

**Comment by harryob** [comment]
#4027

**Pull request closed without merging**